src/shell/lisa_shell

```diff
@@ -93,13 +93,27 @@ fi
 echo -ne "$LISASHELL_DEFAULT"
 }
 
+function _lisa-update-all {
+echo -ne "${LISASHELL_BLUE}"
+echo -e "*** Update LISA installation"
+
+git pull
```

Comment:
[ github ate my reply so resending :cry: ]

I prefer 1). I would do:

1. Abort if there are any uncommitted changes
2. Check that the top-most commit is part of `origin/master`.  If it isn't that means that the user has commits that aren't part of upstream, so we abort.
3. `git pull origin master`.  I went for just `git pull` because I didn't want to make an assumption about the names of the remote but I guess you're right that it's very likely that `origin` points to this repository.

What do you think?